web/ui/static/js/graph/index.js

```diff
@@ -1143,6 +1145,25 @@ function escapeHTML(string) {
   });
 }
 
+function limitMetrics(result) {
+  var MAX_METRICS_NUM = 10;
```

Comment:
Is this the right value?

I'd suggest going with 10k, which is still a reasonable number of metrics to have.

Reply:
@brian-brazil Done.